packages/utils/upgrade/src/modules/version/range.ts

```diff
@@ -19,6 +19,14 @@ export const rangeFromReleaseType = (current: Version.SemVer, identifier: Versio
       const nextMajor = semver.inc(current, 'major') as Version.LiteralSemVer;
       return fromCurrentTo(nextMajor);
     }
+    case Version.ReleaseType.Patch: {
+      const preMinor = semver.inc(current, 'preminor') as Version.LiteralSemVer;
```

Comment:
Versions' tests are currently being developed by @christiancp100 in another branch
@christiancp100 , since your PR will be probably merged after this one, could you add a test for the minor with edge cases like Ben's? (when you'll have access to the actual code)